### PR TITLE
Adding new repository for indexing: rviz_ground_image

### PR DESCRIPTION
**Pull request by Plaba**
# Please Add rviz_static_ground_image to be indexed in the rosdistro.

FOXY

# The source is here:

https://github.com/Plaba/rviz_ground_image.git

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: foxy/distribution.yaml [hunk 1]
```diff
@@ -6221,6 +6221,11 @@ repositories:
       url: https://github.com/ros2/rviz.git
       version: foxy
     status: maintained
+  rviz_static_ground_image:
```
Comment (clalancette) [inline foxy/distribution.yaml:6224]: So there are 3 different names I'm seeing:

The key you are using here is `rviz_static_ground_image`.  The repository name is `rviz-ground-image`.  And the package name in the [package.xml](https://github.com/Plaba/rviz-ground-image/blob/main/package.xml) is `ground_image`.

Let's make all of these the same, preferably with an `rviz` prefix.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: foxy/distribution.yaml (hunk 1 above)
Comment (Plaba) [reply foxy/distribution.yaml]: Thanks for pointing that out! I've renamed everything to `rviz_ground_image`.

**Review by clalancette — APPROVED** [review]
Thanks for doing the renaming, that's helpful.

With that, this looks good to me.

**Comment by Plaba** [comment]
Thanks a lot!

**Pull request merged**